src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -247,24 +247,31 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         switch (toolbarItem.props.name) {
             case "drawRectangle":
                 setSelectionMode(SelectionMode.RECT);
+                this.setEditorMode(EditorMode.Rectangle);
                 break;
             case "drawPolygon":
                 setSelectionMode(SelectionMode.POLYGON);
+                this.setEditorMode(EditorMode.Polygon);
                 break;
             case "drawPolyline":
```

Comment:
We don't currently have toolbar items to draw these things?  We should either remove these or add the toolbar items.  I'm leaning towards removing these since we don't have a clear end to end for how a point or polyline can be used.

Reply:
That makes sense it was just placeholder for now to show what was there